charts/operator/values.yaml

```diff
@@ -31,11 +31,6 @@ role:
       resources:
         - endpoints
         - ingresses
-        - nodes
-        - nodes/metrics
-        - nodes/spec
-        - nodes/stats
-        - nodes/proxy
```

Comment:
I acutally think this will cause an issue, because causes lists out node infor, @danielpanzella we can drop support for this but wanted to raise it